client/components/infinite-list/index.jsx

```diff
@@ -341,6 +328,8 @@ export default class InfiniteList extends React.Component {
 	}
 
 	render() {
+		this.scrollHelper.props = this.props;
+
 		const propsToTransfer = omit( this.props, Object.keys( this.constructor.propTypes ) ),
```

Comment:
There is a cleanup opportunity here: rewrite this expression to avoid `this.constructor.propTypes`. One day we'd like to strip all `propTypes` definitions from the production build and leave them only for development runtime checking. There are handful of place like this that prevent that from happening.

Reply:
Fixed in the latest commit.